folderAWG/Interoperability Precis.md

```diff
@@ -0,0 +1,128 @@
+○ In the interoperable world, creators of conversational agents will be able to choose from multiple conversational platform providers. 
+● Conversational agent: perceived by users to be a single conversation actor. It uses the infrastructure of the conversational platform, and uses one or more conversational capabilities to hold conversations with the user, with continuity of knowledge and persona and a name by which it can be addressed. 
+○ In the interoperable world, there will be both general-purpose and specific-purpose conversational agents. Dialogs, data, context and controls will be passed between agents according to standardized protocols. 
+● Conversational sub-agent: a conversational agent that can only be invoked through another agent. It performs a delegated task on behalf of another conversational agent. Sub-agents have a name and their own discourse context. Sub-agents are sometimes commonly referred to as “skills” or “actions.” 
+○ In the interoperable world, sub-agents will likely emerge as fully realized agents. 
+● Conversational capability: provides specific dialog functions that inform an agent. Capabilities do not have a defined name or voice, nor the continuity of discourse context. A capability may be as limited as a response from an IOT sensor or an information fragment in support of an agent dialog; it may be as expansive as a partial or full dialog on behalf of an agent. 
+○ In the interoperable world, conversational capabilities will exist independently from agents. 
+
+### IV. Why This and Why Now 
+The challenge of the current state for individual and enterprise/organizational users is that the value of voice assistance today is bound by the many limitations of proprietary conversational platforms. In these walled technology gardens: 
+● Users are unable to communicate directly with and through other agents, or access capabilities developed on other platforms. 
+● Users—individual and organizational users alike— are expected to share all data with the platform provider, which could include conversational, tonal, biometric, and biomarker data. 
+● Users must surrender credentials to access in-platform capabilities. 
+● Sub-agent and capability creators and voice innovators are limited to the technology dictates of the proprietary platforms. 
+
+The lack of interoperability between market-leading, proprietary conversational agents requires that organizational users of voice must develop and maintain multiple, proprietary systems, or invest in agent translation and management software. This is among the many reasons a recent European Commission sector inquiry5found that proprietary conversational platforms and their agents limit the market potential and current growth of home-based Internet of Things (IOT) systems. 
```

Comment:
Typo: `inquiry5found`

Reply:
Also, banal, but we say `IOT` above, then here we say `Internet of Things (IOT)`. The latter should probably come first, then "IOT" can be used as an abbreviation from there on out.